src/shared/lib/io/use-events-requests.ts

```diff
@@ -11,23 +11,34 @@ type TUseEventsRequests = () => {
   getEventRestUrl: (param: EventId | undefined) => string
 }
 
-export const useEventsRequests: TUseEventsRequests = () => {
+// TODO: add 403 response handling
+
+export const useEventsRequests: TUseEventsRequests = (token: string | null) => {
```

Comment:
Thought: better to use store to share token and with data instead of using token as parameter for all requests